src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -54,7 +54,7 @@ public static void CheckFileOutOfSourceControl(string file)
 
         public static void AddFileToProject(this Project project, string file, string itemType = null)
         {
-            if (project.IsKind(ProjectTypes.ASPNET_5, ProjectTypes.DOTNET_Core, ProjectTypes.SSDT))
```

Comment:
Not sure what this SSDT project type is, but we only support DotNetCoreWeb, WAPs and Web Sites.